### PR TITLE
Add org.sauerbraten.Sauerbraten

### PR DESCRIPTION
**Pull request by AsciiWolf**

### CODE REVIEW

**Comment by AsciiWolf** [comment]
bot, build org.sauerbraten.Sauerbraten

**Comment by AsciiWolf** [comment]
bot, build org.sauerbraten.Sauerbraten

File: org.sauerbraten.Sauerbraten.json [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+                    "type": "file",
+                    "path": "sauerbraten.desktop"
+                },
+                {
+                    "type": "file",
+                    "path": "sauerbraten.appdata.xml"
+                }
+            ],
+            "build-commands": [
+                "cd src; make",
```
Comment (AsciiWolf) [inline org.sauerbraten.Sauerbraten.json]: Same issue as [with Red Eclipse](https://github.com/flathub/flathub/pull/427#discussion_r194268618).

**Comment by AsciiWolf** [comment]
By the way, I did not submit the AppData and desktop files upstream and did not ask developers if they are interested in co-maintaining the app, because, unfortunately, upstream is quite dead and the last release is from 2013.

**Comment by barthalion** [comment]
Yeah, skimmed over their website, which is why I didn't ask about it here.

**Comment by AsciiWolf** [comment]
bot, build org.sauerbraten.Sauerbraten

**Comment by barthalion** [comment]
Repository has been created: https://github.com/flathub/org.sauerbraten.Sauerbraten

Thanks!

**Pull request closed without merging**